samples/attestation_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.11)
+
+project("Hello World Sample" LANGUAGES C CXX)
+
```

Comment:
sample name needs to be updated

Reply:
I'll update the documentation on the samples once we finalize what we want the APIs to be.